<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>520</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-tic-tac-toe-Desktop_Qt_5_15_2_GCC_64bit-Debug/tic-tac-toe.png</normaloff>../build-tic-tac-toe-Desktop_Qt_5_15_2_GCC_64bit-Debug/tic-tac-toe.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>4500</width>
     <height>520</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 82, 104);
background-color: rgb(82, 150, 173);</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblCurrentPlayer">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Fira Code&quot;;
color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnPos0">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos3">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos6">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnPos1">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos4">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos7">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnPos2">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos5">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPos8">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;
color:white;
background-color: rgb(82, 173, 151);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewGame">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>50</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>410</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(82, 173, 105);
color:white;</string>
       </property>
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
